README.md

```diff
@@ -191,112 +114,107 @@ jsctl operator deploy --credentials /path/to/secret.json --version v0.0.1-alpha.
 
-```shell
-jsctl operator installations apply --cert-manager-replicas 3
-```
+To create an `Installation` resource with jsctl, you can use the `jsctl operator installations apply` command which will apply the generated config to cluster or output it as yaml if `--stdout` flag is passed.
+
+##### Generate base Installation resource
 
-To just output the YAML of the `Installation` resource, provide the `--stdout` flag:
+jsctl can be used as a quickstart config generator for operator's `Installation`
+resource for specific scenarios as an alternative to writing your own
+`Installation` resource from scratch.
 
 ```shell
-jsctl operator installations apply --stdout
+jsctl operator installations apply --stdout > installation.yaml
 ```
+This command generates a base Installation resource that configures the operator to install cert-manager and [approver-policy](https://cert-manager.io/docs/projects/approver-policy/).
 
-##### Install the cert-manager CSI driver
+Take a look at the generated config and see the [Jetstack Secure operator documentation](https://platform.jetstack.io/documentation/reference/js-operator/about) for how to configure additional resources like issuers.
 
-You can also provide the `--csi-driver` flag to include the installation of the [Cert Manager CSI Driver](https://github.com/cert-manager/csi-driver)
-in your cluster:
+Apply the installation to cluster when ready:
 
 ```shell
-jsctl operator installations apply --csi-driver
+kubectl apply -f installation.yaml
 ```
+##### Generate and apply Installation that configures Jetstack Secure components for Venafi TPP user
 
-##### Install the SPIFFE CSI Driver
+jsctl can be used to generate (and optionally apply) operator configuration to set up a cluster with components relevant for Venafi TPP user.
 
-You can also provide the `--csi-driver-spiffe` flag to include the installation of the [SPIFFE CSI Driver](https://github.com/cert-manager/csi-driver-spiffe)
-in your cluster:
+Create a file with Venafi connection details and credentials `connection.yaml`:
 
-```shell
-jsctl operator installations apply --csi-driver-spiffe
+```yaml
+my-default-zone:
+  zone: <tpp-zone>
+  url: <tpp-server-url>
+  # access-token: <access-token could be used instead of username & password>
+  username: <your-username>
+  password: <your-password>
 ```
 
-By default, this deploys the SPIFFE CSI driver in a high-availability configuration, using two replicas. To change the
-number of replicas you can use the `--csi-driver-spiffe-replicas` flag:
+Run:
 
 ```shell
 jsctl operator installations apply \
-  --csi-driver-spiffe \
-  --csi-driver-spiffe-replicas 3
+  --venafi-oauth-helper \
+  --experimental-venafi-issuers="tpp:my-default-zone:foo" \
+  --experimental-venafi-connections-config ./connection.yaml
 ```
 
-##### Install Istio CSR
```

Comment:
I've removed sections that explicitly talk about the flags to generate istio-csr and csi-spiffe configs. The only 'ready' operator config that we can generate now is for Venafi TPP users (because we can only generate TPP issuer config) so I think there are ultimately two use cases where this the `jsctl operator installations apply` is useful- to generate base config that the user then updates with additional issuer config etc and to generate a Venafi TPP-specfic config. They can use the istio-csr and spiffe flags with either of those, but probably not worth to talk about those separately.